### PR TITLE
Turborepo

### PR DESCRIPTION
**Pull request by hubcio2115**
# Breaking Changes
- moved to turborepo
- changed `bun` to `pnpm` because of compatibility issues 

# Important
If after this PR don't pull main to local. Copy your project, move it somewhere. Pull the changes, than copy the project to `apps/web`

### CODE REVIEW

File: .github/workflows/eslint.yml [hunk 1]
```diff
@@ -28,14 +28,9 @@ jobs:
...
       - name: Install Bun
-        run: npm install -g bun
-      - name: Install dependencies and @microsoft/eslint-formatter-sarif
-        run: bun install @microsoft/eslint-formatter-sarif
+        run: npm install -g pnpm
+      - name: Install dependencies
+        run: pnpm install
       - name: Run ESLint
-        run: SKIP_ENV_VALIDATION=true bun lint . --format @microsoft/eslint-formatter-sarif --output-file eslint-results.sarif
+        run: SKIP_ENV_VALIDATION=true pnpm lint
```
Comment (kalisowski) [inline .github/workflows/eslint.yml:35]: why is the microsoft lint formatter gone?

File: apps/web/src/app/layout.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { GeistSans } from "geist/font";
+import { GeistSans } from "geist/font/sans";
```
Comment (kalisowski) [inline apps/web/src/app/layout.tsx:1]: image

File: apps/web/src/components/pricing/premiumPricingCard.tsx [hunk 3]
```diff
@@ -69,7 +69,7 @@ export default function CardStandard({ className, ...props }: CardProps) {
       </CardContent>
 
       <CardFooter>
-        {/* TODO - redirect to shop */}
+        {/* TODO: redirect to shop */}
```
Comment (kalisowski) [inline apps/web/src/components/pricing/premiumPricingCard.tsx:72]: I guess it was changed to work with the TODO addon?

**Review by kalisowski — APPROVED** [review]

**Review by Maxxxxxxxxxxxxxxxxxxxxx — APPROVED** [review]

File: .github/workflows/eslint.yml (hunk 1 above)
Comment (hubcio2115) [reply .github/workflows/eslint.yml:35]: It was needed for sending the output somwhere, idk why I put it there.

**Pull request merged**